### ISSUE TITLE
Indexing DAOs on the web

### ISSUE THREAD

**Issue by listenaddress** [1Hive/apiary#75]
Right now DAOs aren’t discoverable via search engines. I.e. when you search “Aragon Governance DAO”, there's no direct link returned to the DAO or a standard profile page.

We (Open Work Labs + Autark) are starting to work on organization settings and would love to help facilitate data standards and SEO. One thing specifically we’re wondering about is using something like schema.org to help index organization info and tags, and maybe later things like voting events and their associated proposals.

We’re curious to get a sense of the appetite for this. How important is it that DAOs are easily discoverable via web searches? What about the data inside them, like voting events?

**Comment by lkngtn** [comment]
> One thing specifically we’re wondering about is using something like schema.org to help index organization info and tags

I'm not sure I completely follow, could you expand on this idea a bit? 

If there you guys create some sort of metadata within the organization settings thats something we could expose on an Organization Detail page that seems like it would be useful. 

>  maybe later things like voting events and their associated proposals. 

We are planning on creating an "activity feed" #15 and in order to do that we will be analyzing and indexing all events. At a minimum we would be able to pull out the radspec strings to be able to contextualize these events. However, it if there was a richer semantic mapping that may be useful, especially as number of apps and unique events grows, it may be useful to be able to treat events of similar types in specific ways. 

It seems like it would be both interesting and quite useful to have an Organization detail page (possibly pulling data from your Organization Settings work) which includes a feed of all the activity across all of its installed apps. And this would certainly be indexable by search engines and I think help improve organization discovery.

**Comment by onbjerg** [comment]
>It seems like it would be both interesting and quite useful to have an Organization detail page (possibly pulling data from your Organization Settings work)

#46 🙈

**Comment by listenaddress** [comment]
To preface, I'm not an SEO expert and we should definitely run this plan by one before spending too much time. 

> > One thing specifically we’re wondering about is using something like schema.org to help index organization info and tags
>
> I'm not sure I completely follow, could you expand on this idea a bit?
>
> If there you guys create some sort of metadata within the organization settings thats something we could expose on an Organization Detail page that seems like it would be useful.

Yep, this is what I'm suggesting. We could use schema.org standards to create JSON-LD to serve on org detail pages, improving indexability those pages. I'm not sure if we want to store org settings in JSON-LD on our end but it would be trivial to create/cache JSON-LD based on the fields from our org settings.

Here's the Organization schema: https://schema.org/Organization.

Here's a tool for testing tool for JSON-LD with an example with our Organization JSON-LD from openworklabs.com: https://search.google.com/structured-data/testing-tool#url=openworklabs.com. 

So for each DAO we could build an organization JSON-LD object that has selected fields populated from org settings, as well as a `url` field that's pointing to the DAO's mainnet URL.

Does that make sense?

> It seems like it would be both interesting and quite useful to have an Organization detail page (possibly pulling data from your Organization Settings work) which includes a feed of all the activity across all of its installed apps. And this would certainly be indexable by search engines and I think help improve organization discovery.

Agreed! There are definitely a couple options for feed ontologies. Here's one: https://schema.org/DataFeed.

**Comment by onbjerg** [comment]
Makes sense. I haven't quite updated my SEO knowledge since meta tags were the biggest thing, which I understand now is not quite as big, since the search engines rely on other aspects now, so we should definitely run this by someone with more knowledge.

Thank you for the link to the relevant schemas 😊
